src/account/UpgradeableModularAccount.sol

```diff
@@ -571,6 +607,41 @@ contract UpgradeableModularAccount is
     // solhint-disable-next-line no-empty-blocks
     function _authorizeUpgrade(address newImplementation) internal override {}
 
+    function _checkIfValidationApplies(bytes4 selector, FunctionReference validationFunction, bool shared)
+        internal
+        view
+    {
+        AccountStorage storage _storage = getAccountStorage();
+
+        // Check that the provided validation function is applicable to the selector
+        if (shared) {
```

Comment:
this `shared` byte in calldata saves up to 1 `SLOAD`?

Reply:
Yes, that's what it does